### PR TITLE
feat(accessibility): add `isAccessibilityServiceEnabled` section

### PR DESCRIPTION
**Pull request by grgr-dkrk**
This PR adds documentation for the features implemented in https://github.com/facebook/react-native/pull/31396.

### CODE REVIEW

**Review by lunaleaps — APPROVED** [review]
Thanks @grgr-dkrk! Will merge this after the related PR merges!

**Review by Simek — APPROVED** [review]
Since the `0.68` has been released, we can ship this now!

**Pull request merged**
